app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -5,46 +5,56 @@
         android:id="@+id/grp1">
         <item
             android:id="@+id/academics"
-            android:title="@string/academics" />
+            android:title="@string/academics"
+            android:icon="@drawable/ic_academics_black_24px"/>
         <item
             android:id="@+id/admission"
-            android:title="@string/admissions" />
+            android:title="@string/admissions"
+            android:icon="@drawable/ic_admission_black_24px"/>
         <item
             android:id="@+id/placement"
-            android:title="@string/placements" />
+            android:title="@string/placements"
+            android:icon="@drawable/ic_placements"/>
         <item
             android:id="@+id/administration"
-            android:title="@string/administration" />
+            android:title="@string/administration"
+            android:icon="@drawable/ic_administration_ind_black_24px"/>
         </group>
 
     <group android:checkableBehavior="single"
         android:id="@+id/grp2">
         <item
             android:id="@+id/faculty"
-             android:title="@string/faculty" />
+             android:title="@string/faculty"
+            android:icon="@drawable/ic_faculty_black_24px"/>
 
         <item
             android:id="@+id/student"
-            android:title="@string/student" />
+            android:title="@string/student"
+            android:icon="@drawable/ic_student_outline_black_24px"/>
     </group>
 
     <group android:checkableBehavior="single"
         android:id="@+id/grp3">
         <item
             android:id="@+id/emergency"
-            android:title="@string/emergency" />
+            android:title="@string/emergency"
+            android:icon="@drawable/ic_healing_black_24px"/>
 
```

Comment:
you have space in between several segments and not at other places. Please remove space between items under same group.